api/WebSockets.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
I think some of this data belongs in `WebSocket` (it looks as if there's no plural interface). That said, I think the original table contains some questionable data that'd be worth digging into a little more. Perhaps the right thing to do here is to drop this data from the PR entirely and start a separate PR for WebSocket data? That way we don't hold up the rest, which looks pretty good

Reply:
Sure thing!  I'll separate it into a separate PR.  :wink: